src/Tasks/Microsoft.NET.Build.Tasks/RuntimeOptions.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.NET.Build.Tasks
 {
     internal class RuntimeOptions
     {
+        public string tfm { get; set; }
```

Comment:
The caching step is typically done by admin or installer scripts. We do not expect the end user to do this themselves.

When this feature was spec'd, we decided to not do side by side install for the same TFM, to fix the crossgen bug- we would replace the assemblies already cached with the updated assembles, using the above mechanism

The readytorun images are supposed to be resilient to runtime changes  within the same TFM, worst case we are to fallback to jitting

@gkhanna79 : Do correct me if i am wrong